www/index.html

```diff
@@ -197,7 +189,7 @@ <h2>Configuration</h2>
                                 <div id="openLocalFiles" style="display: none;">
                                     Please select the .zim file (or all the .zimaa, .zimab etc in case of a split ZIM file)<br />
                                     <input type="file" id="archiveFiles" multiple class="btn" accept=".zim,.dat,.idx,.txt,.zimaa,.zimab,.zimac,.zimad,.zimae,.zimaf,.zimag,.zimah,.zimai,.zimaj,.zimak,.zimal,.zimam,.ziman,.zimao,.zimap,.zimaq,.zimar,.zimas,.zimat,.zimau,.zimav,.zimaw,.zimax,.zimay,.zimaz,.zimba,.zimbb,.zimbc,.zimbd,.zimbe,.zimbf,.zimbg,.zimbh,.zimbi,.zimbj,.zimbk,.zimbl,.zimbm,.zimbn,.zimbo,.zimbp,.zimbq,.zimbr,.zimbs,.zimbt,.zimbu,.zimbv,.zimbw,.zimbx,.zimby,.zimbz" /><br />
-                                    <strong>Only Wikimedia contents (wiki*.zim* files) have been tested</strong> for now. There are known bugs on other contents, and audio/video/dynamic contents are not supported for now.<br />
+                                    <strong>Only Wikimedia (wiki*.zim* files) and StackExchange contents have been tested</strong> for now. Audio/video/dynamic contents are not supported for now.<br />
```

Comment:
Ditto about potentially adding Wikivoyage to the list.

Reply:
That's why I'm talking about wikiMedia-based ZIM files, not wikiPedia (wiki*.zim* covers wikivoyage).
But I should probably talk about mediawiki (the wiki engine) instead of wikimedia (@kelson42  : a hint on that?)
In any case, I understand it can be confusing, I'd like to find a simpler way to say that all ZIM files based on the same wiki engine as wikipedia should work fine.